{"name": "test_register_success[test_info2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 200 == 2001\n +  where 200 = <Response [200]>.status_code", "trace": "self = <tests.test_register_shop.TestRegister object at 0x000001FDA0DEBD48>, test_info = {'business_license': 'pfeasy/MTYwNDc4MDEwNjAyMS0zNDgzYzI2Nw==.jpg', 'description': '', 'inviter_phone': '', 'latitude': 30.58108413, ...}\nhandler = <middleware.handler.Handler object at 0x000001FDA0DEBC08>\n\n    @pytest.mark.success\n    @pytest.mark.parametrize(\"test_info\", test_data)\n    def test_register_success(self, test_info,handler):\n        easy_session = handler.login_easy()\n        print(test_info)\n        res = easy_session.request(url=url, method=\"post\", json=test_info)\n        print(res.json())\n        logger=handler.logger\n        try:\n            assert res.status_code == 2001\n        except Exception as e:\n            logger.error(\"出错了\")\n>           raise e\n\ntests\\test_register_shop.py:32: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <tests.test_register_shop.TestRegister object at 0x000001FDA0DEBD48>, test_info = {'business_license': 'pfeasy/MTYwNDc4MDEwNjAyMS0zNDgzYzI2Nw==.jpg', 'description': '', 'inviter_phone': '', 'latitude': 30.58108413, ...}\nhandler = <middleware.handler.Handler object at 0x000001FDA0DEBC08>\n\n    @pytest.mark.success\n    @pytest.mark.parametrize(\"test_info\", test_data)\n    def test_register_success(self, test_info,handler):\n        easy_session = handler.login_easy()\n        print(test_info)\n        res = easy_session.request(url=url, method=\"post\", json=test_info)\n        print(res.json())\n        logger=handler.logger\n        try:\n>           assert res.status_code == 2001\nE           assert 200 == 2001\nE            +  where 200 = <Response [200]>.status_code\n\ntests\\test_register_shop.py:29: AssertionError"}, "attachments": [{"name": "log", "source": "dd49b2d2-a350-4507-b6cb-c5d62e33c1dd-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "8248aee0-3211-4f35-95fd-7144c2a17e32-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "634fffb1-da22-4f15-9ce3-4bd12d7262f1-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "test_info", "value": "{'shop_name': '接口测试3', 'shop_img': 'pfeasy/MTYwNDc4MDEwMzgyMi0xYmU2M2ZhYw==.jpg', 'business_license': 'pfeasy/MTYwNDc4MDEwNjAyMS0zNDgzYzI2Nw==.jpg', 'shop_province': 420000, 'shop_city': 420100, 'shop_county': 420106, 'shop_address': 'asdasd', 'latitude': 30.58108413, 'longitude': 114.3162001, 'inviter_phone': '', 'description': ''}"}], "start": 1604862686989, "stop": 1604862687155, "uuid": "994171d5-8fbf-4d3b-bf1e-54f849e0a17e", "historyId": "e6f7f841f4e18fa8de6b7e6e0313e874", "testCaseId": "3c61adbf246aca673030a222b08021ae", "fullName": "tests.test_register_shop.TestRegister#test_register_success", "labels": [{"name": "tag", "value": "success"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_register_shop"}, {"name": "subSuite", "value": "TestRegister"}, {"name": "host", "value": "DESKTOP-5BGKIVL"}, {"name": "thread", "value": "22184-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_register_shop"}]}